nodes/list_masks/mask_join.py

```diff
@@ -87,28 +103,29 @@ def apply_mask(self, mask, data_t, data_f):
         return out
 
     def get_level(self, mask, data_t, data_f, level):
+        if self.choice:
+            apply_mask = self.apply_choice_mask
+        else:
+            apply_mask = self.apply_mask
+            
         if level == 1:
             out = []
-            param = (mask, data_t, data_f)
+            param = list_match_func[self.list_match_global]([mask, data_t, data_f])
             if not all((isinstance(p, (list, tuple)) for p in param)):
                 print("Fail")
```

Comment:
Feel free to remove all prints inside process method of a node.
http://nortikin.github.io/sverchok/docs/contributing/node_api.html#printing-logging